### PR TITLE
Migration/stavevolta

### PR DESCRIPTION
**Pull request by rvilarl**
StaveVolta!

### CODE REVIEW

File: src/stavevolta.ts [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+import { FontInfo } from './types/common';
+import { Vex } from './vex';
 
+export enum VoltaType {
+  NONE = 1,
+  BEGIN = 2,
+  MID = 3,
+  END = 4,
+  BEGIN_END = 5,
+}
```
Comment (0xfe) [inline src/stavevolta.ts:15]: Newline between please.

File: src/stavevolta.ts [hunk 2]
```diff
@@ -1,23 +1,33 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 // Author Larry Kuhns 2011
 
+import { Stave } from './stave';
 import { StaveModifier } from './stavemodifier';
+import { FontInfo } from './types/common';
+import { Vex } from './vex';
```
Comment (0xfe) [inline src/stavevolta.ts:7]: This can be removed.

File: src/stavevolta.ts (hunk 1 above)
Comment (rvilarl) [reply src/stavevolta.ts:15]: done

File: src/stavevolta.ts (hunk 2 above)
Comment (rvilarl) [reply src/stavevolta.ts:7]: done

**Comment by rvilarl** [comment]
Ready

**Comment by 0xfe** [comment]
Looks good. Merging.

**Pull request merged**